database/migrations/2017_06_10_155349_create_currencies_data.php

```diff
@@ -26,6 +26,7 @@ public function up()
       DB::table('currencies')->insert([ 'iso' => 'GBP', 'name' => 'British Pound', 'symbol'=>'£' ]);
       DB::table('currencies')->insert([ 'iso' => 'EUR', 'name' => 'Euro', 'symbol'=>'€' ]);
       DB::table('currencies')->insert([ 'iso' => 'RUB', 'name' => 'Russian Ruble', 'symbol'=>'₽' ]);
+      DB::table('currencies')->insert([ 'iso' => 'BRL', 'name' => 'Brazilian Real', 'symbol'=>'R$' ]);
```

Comment:
Instead of modifying this file, you should use a new migration, like this one (https://github.com/monicahq/monica/blob/master/database/migrations/2017_06_14_132911_add_zar_currency_to_currencies_table.php). If we do like you've done, no existing installations will contain this new currency, because this migration has already been executed.